### DISCUSSION TITLE
Como iniciar na carreira de desenvolvimento backend ? Aprendizados e dicas

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#111] [Talks in Portuguese]
**Palestrante**: Kamila Santos
**Assunto**: Nessa palestra vamos abordar tópicos essenciais para quem deseja se tornar dev backend , o que estudar, como estudar (as técnicas que me ajudam nos estudos), quais assuntos estudar depois de aprender a base, algumas dicas para os processos seletivos, , soft skills e como as comunidades de desenvolvimento podem te ajudar (assim como me ajudaram), nessa jornada.

**Comment by Kamilahsantos** [comment]
Slides : https://speakerdeck.com/kamilahsantos/octogatosconf-como-iniciar-na-carreira-de-desenvolvimento-backend-aprendizados-e-dicas
redes sociais:
https://www.instagram.com/kamila_code/
https://www.youtube.com/Kamilacode
https://www.linkedin.com/in/kamila-santos-oliveira/
https://github.com/Kamilahsantos


Muito obrigada pela oportunidade pessoal :)
